plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CreateAppEngineWtpProjectTest.java

```diff
@@ -0,0 +1,227 @@
+/*
```

Comment:
I created an abstract base class `CreateAppEngineWtpProjectTest` for `CreateAppEngineStandardWtpProjectTest` and `CreateAppEngineFlexWtpProjectTest`, which provides common code and tests that can apply to the both tests. Most of the code was simply moved from the two subclasses for code sharing.